### ISSUE TITLE
Stats Popup: Missing Summary for Unleveled items

### ISSUE THREAD

**Issue by RedHotTrouble** [designker/ishtar-commander-bugs#231]
EDITED: Some* (not all) armor and weapon items whose perks haven't been fully leveled up are missing the "bulleted" summary text in their stats popup.

Device: iPhone 7

Screenshot:
![img_2794](https://cloud.githubusercontent.com/assets/14222975/26815604/780ce0ea-4a4a-11e7-8a72-9b76ecf9d7a2.PNG)

Repro Steps:
1. Inspect item whose perks haven't been fully leveled up.
2. Click on icon for "bulleted" perks summary.
3. In some* cases, bulleted verbiage that shows below item's stats is missing.

**Comment by floatingatoll** [comment]
Verified issue on 848. The selector for perk bubbles still works and correctly shows perk bubbles.

**Comment by floatingatoll** [comment]
On exotics, it shows the text for the exotic peril correctly, but not anything else.

**Comment by RedHotTrouble** [comment]
I just noticed something that may be of interest or help while debugging this issue, or confirming if it's a bug at all. 

In the cases where the bulleted summary is missing, if I click on the area where the verbiage should whow (green question mark in screenshot above), the popup expands and the missing verbiage shows up. 

That's how I was able to realize that this behavior only happens when the bulletted items are more than 2. Amd now I'm not sure if this is actually intended functionality.
